### ISSUE TITLE
New OpenLoco Logo

### ISSUE THREAD

**Issue by Zcooger** [OpenRCT2/OpenLoco#248]
Now I would just need the .png file containing the original silver logo to match it - downsample it, apply some noise and reduce colors.

![open-loco-logo svg](https://user-images.githubusercontent.com/42012614/46908379-2916d500-cf22-11e8-9638-f3f477c1509f.png)

[open-loco-logo.svg.zip](https://github.com/OpenRCT2/OpenLoco/files/2475699/open-loco-logo.svg.zip)

**Comment by AaronVanGeffen** [comment]
It's nice to see someone working on a new logo. While I think the circular shape works fine for an icon, I personally would like the logo to be a little more graphical (e.g. with an actual locomotive). What you've come up reminds me more of a wordmark.

There has been some discussion on [Gitter](https://gitter.im/OpenRCT2/OpenLoco) with regards to a possible logo in the past. It's in the far history of the channel, but perhaps you could join us there for discussion.

**Comment by Zcooger** [comment]
Yes, I tried to remake this rusty gear too, but I need much more time for it. It's already WIP.

**Comment by IntelOrca** [comment]
It is also worth considering how it looks against our sister projects.

OpenTTD | OpenRCT2
------------ | -------------
![openttd](https://upload.wikimedia.org/wikipedia/commons/c/c5/Openttdlogo.svg) | ![openrct2](https://upload.wikimedia.org/wikipedia/commons/0/04/OpenRCT2_logo.svg)

**Comment by Gymnasiast** [comment]
So a square, rotated 45° with a steam locomotive it is, then?

**Comment by IntelOrca** [comment]
Like this in a diamond?
![](http://img.clipartstockpics.com/front-view-thomas-the-train-clipart-byix5om.png)

**Comment by Gymnasiast** [comment]
That image is starting at me.

I don't think it necessarily _has_ to be a steam loc, but it was suggested in the chat (and usually the go-to symbol for a train anyway). I wouldn't mind having the InterCity 125 instead. ;-)

**Comment by Zcooger** [comment]
It's ok?
![openloco_logo svg](https://user-images.githubusercontent.com/42012614/46912517-55107580-cf77-11e8-8c41-417acedb89fe.png)
I will consider it as "icon"
[OpenLoco_logo.svg.zip](https://github.com/OpenRCT2/OpenLoco/files/2476084/OpenLoco_logo.svg.zip)

**Comment by IntelOrca** [comment]
Nice, okay here is my feedback.

* I think the diamond shape is definitely the way forward!
* The icon is perhaps too shiny and 3D in comparison with the other logos.
* A rust coloured train might be better which would mean a different background colour, maybe silver?
* I wonder how a cog pattern (to imitate locomotion's logo) would look on the front of the train.
* I think it would be good to show more of the cow guard if possible.
* A chimney would help identify it more as a steam train.

Thanks. 👍

**Comment by Broxzier** [comment]
I agree with these points, and if it's not too much work for you I'd like to see a version where the cog is in be background between the train and diamond.

**Comment by Zcooger** [comment]
![preview1](https://user-images.githubusercontent.com/42012614/46923363-e4209a80-d016-11e8-9a5b-5afcf95c2e90.png)

**Comment by Zcooger** [comment]
![icon 8](https://user-images.githubusercontent.com/42012614/46939786-7ea8ca00-d067-11e8-9651-25dd912d6d50.png)
![icon 16](https://user-images.githubusercontent.com/42012614/46939787-7ea8ca00-d067-11e8-90b4-ef18153082cb.png)
![icon 32](https://user-images.githubusercontent.com/42012614/46939789-7ea8ca00-d067-11e8-8665-e5a2967c1cfe.png)
![icon 64](https://user-images.githubusercontent.com/42012614/46939790-7ea8ca00-d067-11e8-9fe9-cb1cf1adf1c8.png)
![icon 128](https://user-images.githubusercontent.com/42012614/46939793-7f416080-d067-11e8-966b-584ecd57da0e.png)
![icon 256](https://user-images.githubusercontent.com/42012614/46939794-7f416080-d067-11e8-9ca1-9715637d13d6.png)
![icon 512](https://user-images.githubusercontent.com/42012614/46939795-7f416080-d067-11e8-82ca-04a64f76604a.png)

[OpenLoco Icon.zip](https://github.com/OpenRCT2/OpenLoco/files/2478005/OpenLoco.Icon.zip)

Waiting for more feedback. Two versions, one with steam and without steam (<128px).

**Comment by IntelOrca** [comment]
This looks very good.

* I particularly like the steam coming out of the diamond like the flag did in OpenRCT2.
* The original logo had some green, perhaps that could be used for something in this logo.
* The boiler has a very rounded front when normally the front is flat i.e. a cylinder.
* I think the cog should be removed from 32x32 sizes and below.

**Comment by Zcooger** [comment]
![icon 8](https://user-images.githubusercontent.com/42012614/46948362-1f09e900-d07e-11e8-8876-c5f7d342c6b9.png)![icon 16](https://user-images.githubusercontent.com/42012614/46948363-1f09e900-d07e-11e8-8fcf-48125127a5c0.png)![icon 32](https://user-images.githubusercontent.com/42012614/46948364-1f09e900-d07e-11e8-9522-d455698278bc.png)![icon 64](https://user-images.githubusercontent.com/42012614/46948366-1fa27f80-d07e-11e8-9b47-3b8c87cd766c.png)![icon 128](https://user-images.githubusercontent.com/42012614/46948368-1fa27f80-d07e-11e8-83af-93b99e225a7f.png)![icon 256](https://user-images.githubusercontent.com/42012614/46948369-1fa27f80-d07e-11e8-9146-4ec47557cf16.png)![icon 512](https://user-images.githubusercontent.com/42012614/46948370-203b1600-d07e-11e8-99d3-00518a60de9a.png)
[OpenLoco Icon.zip](https://github.com/OpenRCT2/OpenLoco/files/2478534/OpenLoco.Icon.zip)

**Comment by Zcooger** [comment]
Sure. just add some circles and :tada:
![icon 512](https://user-images.githubusercontent.com/42012614/46955901-51bedc00-d094-11e8-90b3-879bc4bded59.png)
[OpenLoco Icon.zip](https://github.com/OpenRCT2/OpenLoco/files/2479057/OpenLoco.Icon.zip)

**Comment by IntelOrca** [comment]
That looks better, I think the sides as it comes out the chimney need tweaking slightly though:
![image](https://user-images.githubusercontent.com/1482259/46956338-f76e4d00-d08c-11e8-909f-e5b2e56fec00.png)

**Comment by IntelOrca** [comment]
@Zcooger Can you post the photograph of the train you based your logo on?

**Comment by Derpercoaster** [comment]
>     * The boiler has a very rounded front when normally the front is flat i.e. a cylinder.

Actually, the locomotive on the cover of the game (and this logo) does have a rounded front. They were called "Hudsons," and some were streamlined for passenger services. Funnily enough, none of them had the number "4843" (I also think it's odd that there isn't one in the game despite the cover).

**Comment by Derpercoaster** [comment]
> 
> 
> Ok, one more circle: ![icon 512](https://user-images.githubusercontent.com/42012614/46956839-92b7f000-d096-11e8-8b6f-3d5ca1587f9e.png) [OpenLoco Icon.zip](https://github.com/OpenRCT2/OpenLoco/files/2479111/OpenLoco.Icon.zip)

The only real problem I have with this is that the original reference image shows the locomotive from a low perspective, whereas the icon you've made seems to imply a view directly from the front, without an intended perspective. It just looks wrong somehow to me. I think if the headlamp was more centered it would look better.

**Comment by Derpercoaster** [comment]
![image](https://user-images.githubusercontent.com/60452987/139782869-ee80b80a-bcd3-4a8c-9ad4-f4691f13b0d9.png)
This is the best image of the real thing I could find. Shown here, the headlamp is in the center as opposed to near the top. (Also this locomotive looks so... imposing. Powerful, maybe)

**Comment by dschoemehl** [comment]
@duncanspumpkin How do we associate this icon with the .exe? It looks like the .exe is just checked into GitHub and then copied into the corresponding directory.

**Comment by AaronVanGeffen** [comment]
> @duncanspumpkin How do we associate this icon with the .exe? It looks like the .exe is just checked into GitHub and then copied into the corresponding directory.

For Windows, we add a new icon to the openloco.dll file we compile. The .exe is patched to tap into that .dll.

**Comment by dschoemehl** [comment]
OK. Is this supposed to be working? I am still seeing the old icon on my machine.

**Comment by AaronVanGeffen** [comment]
There is no new logo on the title screen yet, if that's what your mean?

**Comment by dschoemehl** [comment]
No, I mean the icon for the .exe
![OpenLoco](https://user-images.githubusercontent.com/4430542/215602550-ef7216df-db2c-4e4e-b12d-e7b53503de85.png)

**Comment by AaronVanGeffen** [comment]
The icon for the .exe hasn't changed, but I think we do ship one in the .dll that should be used while the game is active.

(I'm on Linux so I can't verify)

**Comment by dschoemehl** [comment]
I see. Yeah, the one in the window looks good. 
![image](https://user-images.githubusercontent.com/4430542/215619007-3c0de25b-ad64-47e2-aea9-5b9fdf50b3d4.png)

Would like to fix this situation though since we have such a nice icon to use.
![image](https://user-images.githubusercontent.com/4430542/215619109-48b6917d-cd06-4229-9659-398d86881352.png)

**Comment by AaronVanGeffen** [comment]
> Would like to fix this situation though since we have such a nice icon to use.

You can change the shortcut to use the icon from the .dll instead of the .exe file. Not sure how to do that on modern versions of Windows, though. (Right-click -> properties -> Details -> Change icon? Something like that)

**Comment by dschoemehl** [comment]
Well, that solves the immediate need. Thanks.
![image](https://user-images.githubusercontent.com/4430542/215621052-bb67b3eb-655a-4cab-820c-b0b5c4c41c47.png)

How hard would it be to rebuild the .exe with the proper icon?

**Comment by AaronVanGeffen** [comment]
> How hard would it be to rebuild the .exe with the proper icon?

Not before we've completed the C++ reimplementation. At the moment, our project only builds a .dll file, not a .exe.
